packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -84,6 +84,12 @@ const WorkspaceHeader: React.FC = () => {
   const { stigg } = useStiggContext();
   const { trackEvent } = useTracking();
 
+  const daysLeftText = useMemo(() => {
+    return upgradeButtonData.trialDaysLeft === 1
+      ? `${upgradeButtonData.trialDaysLeft} day left for the free trial`
```

Comment:
instead: `${upgradeButtonData.trialDaysLeft} day${upgradeButtonData.trialDaysLeft !== 1 && "s"} left for the free trial`